# This message contains an array of all currently detected objects.
std_msgs/Header header

# The list of detected objects
DetectedObject[] objects